lib/event_store/supervisor.ex

```diff
@@ -90,7 +91,8 @@ defmodule EventStore.Supervisor do
             Supervisor.child_spec({Registry, keys: :unique, name: subscriptions_registry_name},
               id: subscriptions_registry_name
             ),
-            {Notifications.Supervisor, {name, config}}
+            {Notifications.Supervisor, {name, config}},
+            Telemetry.poller_child_spec(conn: conn, schema: schema)
```

Comment:
❓ Should this poller be opt-in, or opt-out?  It adds load to the DB, so perhaps we should give users the option to enable/disable it.